modules/odc-interacting-with-applications-and-components.adoc

```diff
@@ -16,8 +16,16 @@ This feature is available only when you create applications using the *From Git*
 ====
 +
 * Hover your cursor over the lower left icon on the Pod to see the name of the latest build and its status. The status of the application build is indicated as *New* (image:odc_build_new.png[title="New Build"]), *Pending* (image:odc_build_pending.png[title="Pending Build"]), *Running* (image:odc_build_running.png[title="Running Build"]), *Completed* (image:odc_build_completed.png[title="Completed Build"]), *Failed* (image:odc_build_failed.png[title="Failed Build"]), and *Canceled* (image:odc_build_canceled.png[title="Canceled Build"]).
-* The status or phase of the pod is indicated by different colors and tooltips as *Running* (image:odc_pod_running.png[title="Pod Running"]), *Not Ready* (image:odc_pod_not_ready.png[title="Pod Not Ready"]), *Warning*(image:odc_pod_warning.png[title="Pod Warning"]), *Failed*(image:odc_pod_failed.png[title="Pod Failed"]), *Pending*(image:odc_pod_pending.png[title="Pod Pending"]), *Succeeded*(image:odc_pod_succeeded.png[title="Pod Succeeded"]), *Terminating*(image:odc_pod_terminating.png[title="Pod Terminating"]), or *Unknown*(image:odc_pod_unknown.png[title="Pod Unknown"]).
-For more information about pod status, see the link:https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase[Kubernetes documentation].
+* The status or phase of the pod is indicated by different colors and tooltips as:
+** *Running* (image:odc_pod_running.png[title="Pod Running"]): The Pod has been bound to a node, and all of the containers have been created. At least one container is still running, or is in the process of starting or restarting.
+** *Not Ready* (image:odc_pod_not_ready.png[title="Pod Not Ready"]): In pods that are running multiple containers, not all containers are ready. 
+** *Warning*(image:odc_pod_warning.png[title="Pod Warning"]): Containers in pods must be terminated, however termination did not succeed. Some containers may be other states.
+** *Failed*(image:odc_pod_failed.png[title="Pod Failed"]): All containers in the Pod have terminated, and at least one container has terminated in failure. That is, the container either exited with non-zero status or was terminated by the system.
+** *Pending*(image:odc_pod_pending.png[title="Pod Pending"]): The Pod has been accepted by the Kubernetes cluster, but one or more of the containers has not been set up and made ready to run. This includes time a Pod spends waiting to be scheduled as well as the time spent downloading container images over the network.
+** *Succeeded*(image:odc_pod_succeeded.png[title="Pod Succeeded"]): All containers in the Pod have terminated in success, and will not be restarted.
+** *Terminating*(image:odc_pod_terminating.png[title="Pod Terminating"]): When a Pod is being deleted, it is shown as *Terminating* by some kubectl commands. *Terminating* status is not one of the Pod phases. A Pod is granted a term to terminate gracefully, which defaults to 30 seconds.
```

Comment:
s/term/period
We use the phrase _graceful termination period_ [elsewhere](https://docs.openshift.com/container-platform/4.7/nodes/pods/nodes-pods-priority.html#priority-preemption-graceful_nodes-pods-priority).